libraries/c_sdk/aws/shadow/test/system/aws_iot_tests_shadow_system.c

```diff
@@ -422,6 +429,30 @@ static void _updateGetDeleteBlocking( IotMqttQos_t qos )
 
 /*-----------------------------------------------------------*/
 
+/**
+ * @brief Establish an MQTT connection. Retry if enabled.
+ */
+static IotMqttError_t _mqttConnect( const IotMqttNetworkInfo_t * pNetworkInfo,
+                                    const IotMqttConnectInfo_t * pConnectInfo,
+                                    uint32_t timeoutMs,
+                                    IotMqttConnection_t * const pMqttConnection )
+{
+    IotMqttError_t status = IOT_MQTT_STATUS_PENDING;
+
+    int32_t retryCount = 0;
```

Comment:
Where is this used?

Reply:
Thanks. Should really setup VS to show unused vars.